modules/dns/outputs.tf

```diff
@@ -3,3 +3,8 @@ output "cis_dns_records" {
   description = "DNS records of CIS instance"
   value       = ibm_cis_dns_record.dns_records
 }
+
+output "cis_dns_records_import" {
+  description = "DNS records imported through a file"
+  value       = ibm_cis_dns_records_import.dns_record_import
```

Comment:
I would prefer to name as `cis_imported_dns_records`
Desc: Imported DNS records from a file